src/osd/ECUtil.h

```diff
@@ -32,6 +32,8 @@ const uint64_t CHUNK_INFO = 8;
 const uint64_t CHUNK_PADDING = 8;
 const uint64_t CHUNK_OVERHEAD = 16; // INFO + PADDING
 
+const std::string HINFO_KEY = "hinfo_key";
```

Comment:
> which namespace?

I just mentioned and thought of adding SOME class in:

`namespace ECUtil {`

like:

```
namespace ECUtil {
  class SOME {
<snip>
```

then make ``HINFO_KEY`` private like:

```
namespace ECUtil {
  class SOME {
private
  const std::string HINFO_KEY = "hinfo_key";
<snip>
```

But I stopped myself because it's not going to be reasonably useful with considering modification cost against entire code AND I do not want to mess up anything.